src/toniarts/openkeeper/game/state/MainMenuScreenController.java

```diff
@@ -828,4 +830,76 @@ private void showBriefing() {
             }
         }
     }
+
+    public void showDebriefing(GameResult result) {
+        goToScreen(SCREEN_DEBRIEFING_ID);
+
+        Label levelTitle = screen.findNiftyControl("dLevelTitle", Label.class);
```

Comment:
I get here a NPE. Seems like the screen isn't switched right there. ScreenId says "empty".

Maybe just using `nifty.getScreen(SCREEN_DEBRIEFING_ID)` ?